packages/std/src/mock.rs

```diff
@@ -248,6 +247,7 @@ pub fn mock_ibc_channel_open(
     version: &str,
 ) -> IbcChannelOpenMsg {
     IbcChannelOpenMsg::new(mock_ibc_channel(my_channel_id, order, version))
+        .counterparty_version(version)
```

Comment:
Actually cuz we want this half the time (and not the other half), I would not always set it.

If you take an `initiator: bool` argument in the constructor, you could set/unset this field approriately and also set the intiator bool/enum properly. Seems cleaner than making it and unsetting it in half the tests.

Reply:
(I think this suggestion would obviate the need for `.clear_counterparty_version()` which I suggest above)